isSim: true
hoverHeight: 4.0
descentHeight: 0.3
holdInterval: 5
landHeight: 0.08